### PR TITLE
Items search

### PR DESCRIPTION
**Pull request by kevxo**
Everything is up to date. We need to work on testing to get are endpoints taken care of.

### CODE REVIEW

**Review by Eternal-Flame085 — APPROVED** [review]
looks good

**Review by JoseLopez235 — APPROVED** [review]

File: api/database/models.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Item(db.Model):
     # Auto-incrementing, unique primary key
     id = Column(Integer, primary_key=True)
     # unique item
-    name = Column(String(80), unique=True, nullable=False)
```
Comment (stillsheryl) [inline api/database/models.py:15]: We removed this for the duplicate data, but once we re-classify data we can always add this back.

File: api/resources/items.py [hunk 2]
```diff
@@ -1,24 +1,73 @@
...
+
+@app.route('/api/v1/list/new', methods=['GET'])
+def list():
+      ## not fully done need to talk to the front end and finish passing in the variables
+      data = request.get_json()
+      weathers = data['data']['attributes']['weather']
+      user_categories = data['data']['attributes']['categories']
+      genders = data['data']['attributes']['gender']
+
+      items = db.session.query(Item).filter(
```
Comment (stillsheryl) [inline api/resources/items.py:41]: Great job on the query!

File: api/resources/items.py [hunk 3]
```diff
@@ -1,24 +1,73 @@
...
+      for item in items:
+        if item.category in categories:
+          categories[item.category].append(_item_payload(item))
+        else:
+          categories[item.category] = []
+          categories[item.category].append(_item_payload(item))
+
+      category_obj = {
+        "data": {
+        "id": datetime.now().strftime('%Y%m-%d%H-%M%S-') + str(uuid4()),
```
Comment (stillsheryl) [inline api/resources/items.py:57]: Unique ID created with date

File: manage.py [hunk 4]
```diff
@@ -5,6 +5,8 @@
 from api.database.models import Item
 from tests import db_drop_everything
 
+import psycopg2
```
Comment (stillsheryl) [inline manage.py:8]: Needed for the command to seed the database with the CSV files

File: manage.py [hunk 5]
```diff
@@ -23,28 +25,13 @@ def db_seed():
...
-    db.session.add(item2)
-    db.session.add(item3)
-    db.session.add(item4)
-    db.session.add(item5)
-    db.session.add(item6)
-    db.session.add(item7)
-    db.session.add(item8)
-    db.session.add(item9)
-    db.session.add(item10)
+    conn = psycopg2.connect("host=localhost dbname=pack-smart-dev user=postgres")
```
Comment (stillsheryl) [inline manage.py:28]: We change this from `-dev` and `-test` when seeding each db

**Review by stillsheryl — APPROVED** [review]
Nice!

**Pull request merged**